### PR TITLE
Added close/open button to details page of milestone

### PR DESCRIPTION
**Pull request by BElluu**
New feature #19083

Depending on the current state, a close or open button appears.

--
image
image
--

### CODE REVIEW

File: templates/repo/issue/milestone_issues.tmpl [hunk 1]
```diff
@@ -9,6 +9,13 @@
 			{{if not .Repository.IsArchived}}
 				<div class="column right aligned">
 					{{if or .CanWriteIssues .CanWritePulls}}
+						{{if .Milestone.IsClosed}}
+							<a class="ui green basic button link-action" href data-url="{{$.RepoLink}}/milestones/{{.Milestone.ID}}/open">{{$.locale.Tr "repo.milestones.open"}}
```
Comment (zeripath) [inline templates/repo/issue/milestone_issues.tmpl]:
```suggestion
							<a class="ui green basic button link-action" href data-url="{{$.RepoLink}}/milestones/{{.MilestoneID}}/open">{{$.locale.Tr "repo.milestones.open"}}
```

File: templates/repo/issue/milestone_issues.tmpl [hunk 2]
```diff
@@ -9,6 +9,13 @@
...
+							</a>
+						{{else}}
+							<a class="ui red basic button link-action" href data-url="{{$.RepoLink}}/milestones/{{.Milestone.ID}}/close">{{$.locale.Tr "repo.milestones.close"}}
```
Comment (zeripath) [inline templates/repo/issue/milestone_issues.tmpl]:
```suggestion
							<a class="ui red basic button link-action" href data-url="{{$.RepoLink}}/milestones/{{.MilestoneID}}/close">{{$.locale.Tr "repo.milestones.close"}}
```

**Review by zeripath — APPROVED** [review]
o/w LGTM

File: templates/repo/issue/milestone_issues.tmpl (hunk 2 above)
Comment (BElluu) [reply templates/repo/issue/milestone_issues.tmpl]: Changed :)

File: templates/repo/issue/milestone_issues.tmpl (hunk 1 above)
Comment (BElluu) [reply templates/repo/issue/milestone_issues.tmpl]: Changed :)

**Review by 6543 — APPROVED** [review]

**Pull request merged**